s3_resource/src/s3_operations.cpp

```diff
@@ -68,6 +70,15 @@ using s3_transport_config = irods::experimental::io::s3_transport::config;
 
 namespace irods_s3 {
 
+    inline static const std::string  SHARED_MEMORY_KEY_PREFIX{"irods_s3-shm-"};
+    inline static const int          DEFAULT_SHARED_MEMORY_TIMEOUT_IN_SECONDS{180};
```

Comment:
The use of `const` can be replaced with `constexpr`.

Reply:
I did except wasn't able to get it to work with std::string.  Seems like I would need a string_view.